tox.ini

```diff
@@ -29,11 +29,10 @@ description =
     alldeps: with all optional dependencies
     devdeps: with the latest developer version of key dependencies
     oldestdeps: with the oldest supported version of key dependencies
-    cov: and test coverage
 setenv =
     MPLBACKEND = agg
     COLUMNS = 180
-    PYTEST_COMMAND = pytest -vvv --pyargs sunpy --cov=sunpy --cov-config={toxinidir}/setup.cfg {toxinidir}/docs
+    PYTEST_COMMAND = pytest -vvv --pyargs sunpy --cov-report= --cov=sunpy --cov-config={toxinidir}/setup.cfg {toxinidir}/docs
```

Comment:
This should suppress the cov report output in all our CI.

Reply:
This hasn't broken codecov right?

Reply:
Still reporting ok.